packages/app-store/googlecalendar/api/callback.ts

```diff
@@ -95,10 +96,9 @@ async function getHandler(req: NextApiRequest, res: NextApiResponse) {
     }
 
     // Only attempt to update the user's profile photo if the user has granted the required scope
-    // TODO: Use the avatarUrl when setting the profile picture
```

Comment:
@joeauyeung commented this out in #22074 to stop creating buggy records in DB. I am enabling it back in this PR since we are putting out a fix.